librz/asm/arch/luac/v53/analysis_53.c

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// SPDX-FileCopyrightText: 2017 pancake <pancake@nopcode.org>
+// SPDX-FileCopyrightText: 2018 Maijin
```

Comment:
Not sure if i deserve to be here tbh - i only changed a prefix

Reply:
I guess that's your right to get your name removed :D

Reply:
> Please remove the lines with my name in these, I didn't do anything in that lua stuff - pointless imho/ @ret2libc unless that's required?

Sorry for my late response since I was preparing for other thing these days, I will remove it in the next push :D

Reply:
thanks